tests/test_modelutils.py

```diff
@@ -0,0 +1,103 @@
+import logging
+import os
+import unittest
+
+import astropy.units as u
+import numpy as np
+
+import pint.models.model_builder as mb
+import pint.toa as toa
+import test_derivative_utils as tdu
```

Comment:
Same for this one.

Reply:
Now I'm a bit confused about which imports you're referring to, but I removed `tdu` since it is definitely not used.